### PR TITLE
Add global user age as a feature

### PR DESCRIPTION
**Pull request by Ladsgroup**

### CODE REVIEW

File: revscoring/datasources/types.py [hunk 1]
```diff
@@ -66,3 +66,9 @@ def __init__(self, id=None, name=None, editcount=None, registration=None,
             if block_reason is not None else None
         self.block_expiry = str(block_expiry) \
             if block_expiry is not None else None
+
+        if globalinfo is None:
```
Comment (halfak) [inline revscoring/datasources/types.py]: We should probably store globalinfo.  That way, other features can make use of it.  We really shouldn't be figuring out "globalage" within the datasource itself.  I think the datasource should just provide the raw data and features can figure out how they'll make use of that data.

File: revscoring/extractors/api.py [hunk 2]
```diff
@@ -239,6 +239,12 @@ def query_users_by_text(self, user_texts, batch=50, **params):
                                        ususers=batch_texts, **params)
 
                 for user_doc in doc['query'].get('users', []):
```
Comment (halfak) [inline revscoring/extractors/api.py]: Why not make global_user_info it's own thing?  Maybe having it be a subquery within user_info is unnecessary.

**Comment by halfak** [comment]
@Ladsgroup, it looks like this branch got overwritten.  Is that intentional?

**Comment by Ladsgroup** [comment]
No, I actually made a mess...

**Comment by halfak** [comment]
I'm going to close this since it's just clutter at the moment.

**Pull request closed without merging**